services/s3control/src/main/resources/codegen-resources/customization.config

```diff
@@ -1,6 +1,7 @@
 {
     "serviceSpecificClientConfigClass": "S3ControlConfiguration",
     "serviceConfigHasDualstackConfig": true,
+    "serviceConfigHasFipsConfig": true,
```

Comment:
Have we considered changing `serviceSpecificClientConfigClass` to an object and making  `serviceConfigHasFipsConfig` and `serviceConfigHasDualstackConfig` its fields because they seem to only apply to services that have service config

Reply:
That likely wouldn't break anyone. I'd be surprised if anyone is using serviceSpecificClientConfigClass outside of us, and code search seems to support that. I'll make that change.